src/Evolution/DiscontinuousGalerkin/Actions/ComputeTimeDerivative.hpp

```diff
@@ -491,7 +559,7 @@ ComputeTimeDerivative<Dim, EvolutionSystem, DgStepChoosers, LocalTimeStepping>::
           //  - evolution::dg::Tags::NormalCovectorAndMagnitude<Dim>,
           //  - evolution::dg::Tags::MortarData<Dim>
           detail::internal_mortar_data<EvolutionSystem, Dim>(
-              make_not_null(&box),
+              make_not_null(&box), make_not_null(&face_temporaries),
```

Comment:
If you decide to pass the pointer, here it would be a `make_not_null(&buffer[(VarsTemporaries::number_of_independent_components + ...])`